test/parallel/test-net-socket-write-after-close.js

```diff
@@ -12,7 +12,7 @@ const net = require('net');
       client.on('error', common.mustCall((err) => {
         server.close();
         assert.strictEqual(err.constructor, Error);
-        assert.strictEqual(err.message, 'write EBADF');
+        assert.strictEqual(err.message, 'read EINVAL');
```

Comment:
This is because the write error is now 2 ticks away and the read error comes before